portfolio_website.html

```diff
@@ -17,44 +18,193 @@
+              <div class="proj-txt">
+                <h3 class="proj-head">Professional Art Printing Data</h3>
+                <p class="proj-para">
+               A daily selection of privately personalized reads; no accounts or
+               sign-ups required. has been the industry's standard dummy text ever
+               since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+               <ul class="flex project btn-flex">
+                  <li class="proj-btn">html</li>
+                  <li class="pproj-btn">bootstrap</li>
+                  <li class="proj-btn">Ruby</li>
+               </ul>
+              </div>
+              <a href="/" class="cursor project-link">See Project</a>
+             </li>
+             <li class="proj-con">
+              <div class="proj-txt">
+                <h3 class="proj-head">Professional Art Printing Data</h3>
+                <p class="proj-para">
+               A daily selection of privately personalized reads; no accounts or
+               sign-ups required. has been the industry's standard dummy text ever
+               since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+               <ul class="flex project btn-flex">
+                  <li class="proj-btn">html</li>
+                  <li class="pproj-btn">bootstrap</li>
+                  <li class="proj-btn">Ruby</li>
+               </ul>
+              </div>
+              <a href="/" class="cursor project-link">See Project</a>
+            </li>
+            <li class="proj-con">
+              <div class="proj-txt">
+                <h3 class="proj-head">Professional Art Printing Data</h3>
+                <p class="proj-para">
+               A daily selection of privately personalized reads; no accounts or
+               sign-ups required. has been the industry's standard dummy text ever
+               since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+               <ul class="flex project btn-flex">
+                  <li class="proj-btn">html</li>
+                  <li class="proj-btn">bootstrap</li>
+                  <li class="proj-btn">Ruby</li>
+               </ul>
+              </div>
+              <a href="/" class="cursor project-link">See Project</a>
+           </li>
+            </ul>
+          </section>
+          <section class="about-section">
+            <h2 class="greetings about">About me</h2>
+            <div class="about-container img2">
+              <p class=" intro">
+                Hello I'm a software developer! I can help you build a product,
+                feature or website. Look through some of my work and experience! If
+                you like what you see and have a project you need coded, dont hestiate
+                to contact me.
+              </p>
+              <a class="btn" href="#">Get My Resume</a>
+              <p class="underline"></p>
```

Comment:
Kindly change the p tag to a button and decrease the space between the paragraph and the below section